src/components/NavBar/NavBar.jsx

```diff
@@ -0,0 +1,115 @@
+
+import {Link} from "react-router-dom";
+import {useAuth} from "../../context"
+const NavBar = (props) => {
+
+    const {authState} = useAuth();
+    return (
+        <nav className="nav nav-shadow navbar-rp">
+            <ul className="rui-drawer-content--list no-list">
+                <li
+                    className="flex-row justify-content-space-between align-center rui-drawer-content rui-drawer-header display-inactive">
+                    <section>
+                        <img src="https://raw.githubusercontent.com/sonishreyas/rippleUI/dev/components/media/images/ripple-logo.png"
+                            alt="Logo of ripple play" className="brand-logo" />
+                        <sub className="brand-name">Ripple Play</sub>
+                    </section>
+                    <section>
+                        <i className="fas fa-angle-left close-drawer"></i>
+                    </section>
+                </li>
+                <li className="rui-drawer-content m-10">
+                    <Link to={"/"} className="rui-drawer-links">
+                        <section className={`navbar-rp-section ${ props.home ? "text-cta-color": ""}`}>
```

Comment:
[`By default, an active class is added to a <NavLink> component when it is active.`](https://reactrouter.com/docs/en/v6/api#navlink). So you can use that instead of using prop to show the active state.